src/FsAutoComplete.Core/TipFormatter.fs

```diff
@@ -1041,39 +101,83 @@ let private buildFormatComment cmt (formatStyle : FormatCommentStyle) (typeDoc:
     | _ -> ""
 
 
-let private formatGenericParamInfo cmt =
-  let m = Regex.Match(cmt, """(.*) is (.*)""")
-  if m.Success then
-    sprintf "* `%s` is `%s`" m.Groups.[1].Value m.Groups.[2].Value
-  else
-    cmt
-
-
-let formatTip (FSharpToolTipText tips) : (string * string) list list =
+let private formatGenericParamInfo (text: TaggedText []) =
```

Comment:
definitely not final, lots of stuff has changed in the TaggedText/Tooltip APIs and we'll need to do work here to adjust